lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -41,16 +41,16 @@ namespace {
 //                   Functions for executing code using JIT
 //===----------------------------------------------------------------------===//
 
-/// Emit the entry point for JIT called "jitmain". It simply calls the main
-/// entry of the module with the constant concrete addresses of all the memory
-/// areas. Since these addresses are constants, the LLVM optimizer will constant
-/// propagate them into relative addressing computations and the like and
-/// produce a very efficient code that uses absolute addressing whenever
-/// possible.
+/// Emit the entry point for JIT called "jitmain".
+/// Function has the following API:
+/// void jitmain(uint8_t *baseConstantWeightVars, uint8_t *baseInoutWeightVars,
```

Comment:
Can you add two spaces between the comment marker and the "void" to make sure that the code looks like doc and not some commented out code? I am not sure if clang-format will allow this.